src/llm_vm/onsite_llm.py

```diff
@@ -467,3 +515,36 @@ def finetune(self, dataset, optimizer, c_id):
         # optimizer.storage.set_training_in_progress(c_id, False)
         # if old_model is not None:
         #     openai.Model.delete(old_model)
+
+
+# LLM Quantization
+@RegisterModelClass("quantized")
+class Quantized_Onsite_LLM(Base_Onsite_LLM):
+    model_id = "facebook/opt-350m"  # Store the model ID
```

Comment:
the way this is written its hard_coded to just this url right?

Reply:
Yes, the model_id is currently hard-coded. But I can update it according to your suggested edit.